addons/knowledge/static/src/components/emoji_picker/emoji_picker.js

```diff
@@ -0,0 +1,62 @@
+/** @odoo-module **/
+
+import emojis from '@mail/js/emojis';
+const { Component, useSubEnv } = owl;
+
+class Emoji extends Component {
+    /**
+     * @override
+     */
+    setup () {
+        super.setup();
+        // Mock the template variables:
+        this.className = '';
+        this.emojiListView = {
+            /**
+             * @param {Event} event
+             */
+            onClickEmoji: event => {
+                const $target = $(event.target);
```

Comment:
no need for jquery: use native js code: `event.target.dataset.unicode`. see https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset (jquery is deprecated, new code should not use it)